setup.py

```diff
@@ -2,51 +2,6 @@
 
 import os
 
-from setuptools import find_packages, setup
+import setuptools
 
-
-def get_requirements():
-    with open("./requirements.txt") as file:
-        return file.readlines()
-
-# https://packaging.python.org/guides/single-sourcing-package-version/
-version = {}
-with open("./ansible_bender/version.py") as fp:
-    exec(fp.read(), version)
-
-long_description = ''.join(open('README.md').readlines())
-
-
-setup(
-    name='ansible-bender',
-    version=version["__version__"],
-    description="A tool which builds container images using Ansible playbooks.",
-    long_description=long_description,
-    # long_description_content_type='text/markdown',
-    packages=find_packages(exclude=['tests']),
-    python_requires='>=3.6',
-    install_requires=get_requirements(),
-    entry_points='''
-        [console_scripts]
-        ab=ansible_bender.cli:main
-    ''',
-    license='MIT',
-    classifiers=[
-        'Development Status :: 4 - Beta',
-        'Environment :: Console',
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: MIT License',
-        'Operating System :: POSIX :: Linux',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
-        'Topic :: Software Development',
-        'Topic :: Utilities',
-    ],
-    keywords='containers,ansible,buildah',
-    author='Tomas Tomecek',
-    author_email='tomas@tomecek.net',
-    url='https://github.com/TomasTomecek/ansible-bender',
-    include_package_data=True,
-)
+setuptools.setup(use_scm_version=True)
```

Comment:
Also verify `pip install --no-binary :all:` to ensure it works well with sdists.

Reply:
That sounds like an overkill to me: if it installs as sdist, wheel should be fine, right? Especially when my package doesn't have any extensions or no creepy stuff.

Reply:
Yep, that command above is actually checking for sdist :) It tells pip to avoid any wheels. (it's still missing the last arg though).

sdist is installed by running `setup.py` on user's machine, that's why it's important to check it. wheels installation is just about unpacking zip, no arbitrary code is run. that's why it's fine to skip checking it :)

Reply:
I'm doing that with `pip3 install .` -- my expectation is that's sdist.

Reply:
okay, that's reasonable